webpack.config.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+const NodePolyfillPlugin = require("node-polyfill-webpack-plugin");
+
+/** @type {import("webpack").Configuration} */
+module.exports = {
+    mode: "none",
+    entry: {
+        no_unsanitize: ["./web.js"],
+    },
+    output: {
+        filename: "[name].js",
+        library: "[name]",
+        libraryTarget: "umd",
+        globalObject: "this",
+    },
+    module: {
+        rules: [
+            {
+                test: /\.m?js$/u,
+                loader: "babel-loader",
+                options: {
+                    presets: [
+                        ["@babel/preset-env", {
+                            debug: true, // ← to print actual browser versions
+
+                            /*
+               * We want to remove `transform-unicode-regex` convert because of https://github.com/eslint/eslint/pull/12662.
+               *
+               * With `>0.5%`, `@babel/preset-env@7.7.6` prints below:
+               *
+               *     transform-unicode-regex { "chrome":"49", "ie":"11", "safari":"5.1" }
+               *
+               * So this excludes those versions:
+               *
+               * - IE 11
+               * - Chrome 49 (2016; the last version on Windows XP)
+               * - Safari 5.1 (2011-2013; the last version on Windows)
+               */
+                            targets: ">0.5%, not chrome 49, not ie 11, not safari 5.1",
+                        }],
+                    ],
+                },
+            },
+        ],
+    },
+    plugins: [
+        new NodePolyfillPlugin(),
+    ],
+    resolve: {
+        mainFields: ["browser", "main", "module"],
```

Comment:
If I'm not mistaken [resolve.mainFields](https://webpack.js.org/configuration/resolve/#resolvemainfields) should be instructing webcompat about which package.json property to prefer when importing dependency into a webcompat bundle, and so even if we don't expose any of those properties (and I'm not sure it does make any difference for the bundle generated from this repo at the moment) it may be worth leaving a comment to explain what it does and don’t mistake it as related to the package.json in this repo (where we only export a main property and none of the others).

It may not be doing much at the moment but I think it may be reasonable to keep it given that it doesn’t require a new dependency but just webpack configuration